R/fit.R

```diff
@@ -273,10 +274,15 @@ mmrm <- function(formula,
                  reml = TRUE,
                  optimizer = "automatic",
                  n_cores = 1L,
-                 accept_singular = TRUE) {
+                 accept_singular = TRUE,
+                 method = c("Satterthwaite", "Kenward-Roger", "Kenward-Roger-Linear", "ST", "KR", "KRLin")) {
   assert_string(optimizer)
   use_automatic <- identical(optimizer, "automatic")
-
+  method <- match.arg(method)
+  method <- h_get_method(method)
```

Comment:
I would avoid this extra logic - either go for long names or for short names, but not allow both
```suggestion
```